packages/firebase/index.d.ts

```diff
@@ -5434,6 +5434,88 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List all items (files) and prefixes (folders) under this storage reference.
+     *
+     * It is a helper method for calling list() recursively until there is no
+     * more results. The default pagination size is 200.
```

Comment:
Why do we not use the maximum page size? It seems like it would be faster (less roundtrips) and less likely to be inconsistent.